src/main/java/org/zalando/nakadi/security/AuthorizedClient.java

```diff
@@ -0,0 +1,28 @@
+package org.zalando.nakadi.security;
+
+import javax.annotation.concurrent.Immutable;
+import java.util.Collections;
+import java.util.Set;
+
+@Immutable
+public final class AuthorizedClient implements Client {
+
+    private final String clientId;
+    private final Set<String> scopes;
+
+    public AuthorizedClient(final String clientId, final Set<String> scopes) {
+        this.clientId = clientId;
+        this.scopes = scopes;
+    }
+
+    @Override
+    public boolean is(final String clientId) {
+        return this.clientId.equals(clientId);
+    }
+
+    @Override
+    public Set<String> getScopes() {
```

Comment:
I understood that the original idea of having this class hierarch (Client/Authorized) was to use polymorphism (method `is`) and delegate some logic to this object.

This property accessor `getScopes` contains no logic. I would expect something like `containsScopes` as another "check" method that returns boolean. It would feel more homogeneous approach. 

It would also remove the need for a static class like the `ScopeHelper`.

What do you think?